src/common/exif.cc

```diff
@@ -308,11 +309,11 @@ static const char *_exif_get_exiv2_tag_type(const char *tagname)
   {
     char *t = (char *)tag->data;
     if(g_str_has_prefix(t, tagname) && t[strlen(tagname)] == ',')
```

Comment:
I would use `{ }` around the below `if`. I understand you've only reformatted, but this is frankly horrible code, better fix it now.

Reply:
Sure, it also looks like pretty horrible code to me